CHANGELOG.asciidoc

```diff
@@ -79,6 +79,8 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 - Fix Kubernetes calculated fields store. {pull}6564{6564}
 - Exclude bind mounts in fsstat and filesystem metricsets. {pull}6819[6819]
 - Don't stop Metricbeat if aerospike server is down. {pull}6874[6874]
+- Fix Kubernetes calculated fields store. {pull}6564[6564]
```

Comment:
Seems like that line of changelog sneaked in?

Reply:
Umm, I don't know why this happens :? removing it

Reply:
rebase with 3 way merges. Unfortunately a common issue.